### ISSUE TITLE
Change term - acceptedNameUsage

### ISSUE THREAD

**Issue by ianengelbrecht** [tdwg/dwc#515]
The comments for acceptedName usage refer to a term, dwc:accordingTo, which just needs to be updated to dwc:nameAccordingTo.

**Comment by sharifX** [comment]
Hi,  This proposal is about the explanatory comments that appear in the data — not about changing the term itself. Is this correct? The term _dwc:acceptedNameUsage_ stays the same, but the comment refers to _dwc:accordingTo_, and that should be updated to _dwc:nameAccordingTo_. Does this change impact _dwc:acceptedNameUsageID_? Can you add an example? thanks.

**Comment by tucotuco** [comment]
@sharifX Correct, this change is simply to correct an error in the usage comments.

Adding an example is not a simple erratum. It would require a new term change request to be made with a proposal for the addition of an example and public review in its own right.
